egs/csj/asr1/run.sh

```diff
@@ -48,11 +50,15 @@ maxlen_out=150 # if output length > maxlen_out, batchsize is automatically reduc
 opt=adadelta
 epochs=15
 
+# rnnlm related
+lm_weight=0.3
```

Comment:
is this value (0.3) obtained by testing several values?
If so, it is fine.
If not, you might test several values.

Reply:
Yes, I have tried {0.25, 0.3, 0.5, 1.0}, and 0.3 is the best.